src/olympia/amo/tests/ui/setup.cfg

```diff
@@ -0,0 +1,4 @@
+[pytest]
+addopts = -r a --verbose --junit-xml=results.xml --html=results.html
+base_url = https://addons-dev.allizom.org
+sensitive_url = mozilla\.(com|org)
```

Comment:
There's an issue with merging these files due to test collection and packages, but I'll revisit it and at least determine the variables causing me problems so we can try to work through them.